src/components/NoteInput/NoteInput.scss

```diff
@@ -67,6 +67,16 @@ $note-input__input-right: 40px;
   transition: all 0.08s ease-out;
 }
 
+.toast-button {
+  background-color: $color-backlog-blue;
+  border-style: none;
+  border-radius: 12px;
+  height: 40px;
+  width: 80%;
+  margin: 10px;
```

Comment:
Please use the `$margin--` when you can.
Also I think the horizontal margin looks out of place. Maybe remove it, or center the button.

<img width="349" alt="image" src="https://user-images.githubusercontent.com/44020029/198977728-0d03714e-dc9a-4689-ae10-8ec3bbbe1e19.png">

<img width="355" alt="image" src="https://user-images.githubusercontent.com/44020029/198977841-9b25fee5-296c-4aba-a87c-040c4d03a13a.png">

Reply:
I removed the horizontal margin and tried different alignments and widths. I think, I would go with the 1st one.

![100%width](https://user-images.githubusercontent.com/58441634/199478051-1b81c034-4178-4e04-a73a-7db35a27e5ae.png)
![70%width-centered](https://user-images.githubusercontent.com/58441634/199478019-7538729a-8dc0-42cf-884c-2cdf475b32f8.png)
![fit-content](https://user-images.githubusercontent.com/58441634/199478080-6677e624-2dfc-491f-86b9-0ab84828dbad.png)

Reply:
I like the second one too, and in my opinion the third one just misses some padding to the sides, but this is completely your choice which to choose :)